drivers/hmis/spec/requests/hmis/lookup_client_spec.rb

```diff
@@ -175,6 +176,55 @@
     expect(result.dig('data', 'client', 'image')).not_to be_nil
   end
 
+  context 'with a client who is chronically homeless per HUD definition' do
```

Comment:
@gigxz I'd like to pair with you on this to get a better understanding... I'm copying here from the hud chronic spec (https://github.com/greenriver/hmis-warehouse/blob/5eed1631d1ae87951eb01bb1b4d4e7d064142b27/spec/models/grda_warehouse/hud_chronic_spec.rb#L4). I don't think we want to be invoking all these GrdaWarehouse factories for a hmis request spec test, but I don't know how to properly set up the models so that the warehouse client is considered chronic.

I am also stuck on doing this in my local (not spec) environment. I created a local client using the PIT and set their disability and months homeless values such that I would expect them to be chronic, but it's not the case.

Reply:
Ok! happy to pair. I may have answered the first part of your question here https://github.com/greenriver/hmis-warehouse/pull/4809/files#r1792473983, I agree we should avoid the grda factories.

Reply:
Oof, okay, I got there in the end! My local PIT project had the wrong type.